src/Raven.Server/Dashboard/DatabasesInfoNotificationSender.cs

```diff
@@ -97,6 +98,15 @@ private class DatabaseInfoCache
             public DateTime NextDiskSpaceCheck;
         }
 
+        private static readonly SystemInfoCache CachedSystemInfo = new SystemInfoCache();
```

Comment:
Why have this?
Note that in some cases, we might have these things come and go dynamically.
I'm also concerned about `MountPoints` being a static `List<T>`, which can cause concurrency issues

Reply:
Code refactored - See latest commit